metricbeat-tests/features/apache.feature

```diff
@@ -4,9 +4,11 @@ Feature: As a Metricbeat developer I want to check that the Apache module works
 Scenario Outline: Check module is sending metrics to Elasticsearch without errors
   Given Apache "<apache_version>" is running for metricbeat
     And metricbeat is installed and configured for Apache module
+    And metricbeat waits "20" seconds for the service
```

Comment:
Would it be possible to use containers health report instead of this?

Reply:
Adding a healthcheck at compose level could help I guess. On the other hand, testcontainers supports creating services with WaitFor strategies (https://github.com/testcontainers/testcontainers-go/tree/master/wait)

Reply:
My concern with both `when` clauses is the same: the more we push on arbitrary timings the more we will need to keep adjusting tests later. Also about how users could define the conditions for a healthy scenario, if we take them out of `docker-compose` that means we cannot reuse the scenario in current "unit" tests.

In general I would expect this to work without new parameters, but configuration on health checks.

Reply:
Yeah, probably the `Given Apache "<apache_version>" is running for metricbeat` should be enough to ensure that Apache is running.

Reply:
I can confirm that after using the new observability-ci images (generated from Beats repo, which include HEALTCHECK's) the MySQL tests are passing, because the HEALTHCHECK is present in the Dockerfiles